### PR TITLE
move pip bootstrap into a seperate class

### PR DESCRIPTION
**Pull request by feltra**

### CODE REVIEW

**Comment by feltra** [comment]
@bastelfreak just fixed previous errors and followed your recommendations.

File: manifests/install.pp [hunk 1]
```diff
@@ -72,19 +72,8 @@
...
-      }
-
-      # Puppet is opinionated about the pip command name
-      file { 'pip-python':
-        ensure  => link,
-        path    => '/usr/bin/pip-python',
-        target  => '/usr/bin/pip',
-        require => Exec['bootstrap pip'],
+      class { 'python::pip::boostrap':
+        version => 'pip',
```
[the remark this answers is not in the document]
Comment (feltra) [reply manifests/install.pp]: We can skip it, but this is to enforce installing pip instead of pip3.

File: manifests/install.pp [hunk 2]
```diff
@@ -72,20 +72,7 @@
...
-      }
-
-      # Puppet is opinionated about the pip command name
-      file { 'pip-python':
-        ensure  => link,
-        path    => '/usr/bin/pip-python',
-        target  => '/usr/bin/pip',
-        require => Exec['bootstrap pip'],
-      }
+      class { 'python::pip::boostrap':; }
```
Comment (bastelfreak) [inline manifests/install.pp]: if you don't want to provide any value, please use the include syntax:
```suggestion
      include 'python::pip::boostrap'
```

File: manifests/install.pp [hunk 3]
```diff
@@ -72,20 +72,7 @@
...
-      }
-
-      # Puppet is opinionated about the pip command name
-      file { 'pip-python':
-        ensure  => link,
-        path    => '/usr/bin/pip-python',
-        target  => '/usr/bin/pip',
-        require => Exec['bootstrap pip'],
-      }
+      include 'python::pip::boostrap'
```
Comment (bastelfreak) [inline manifests/install.pp]: ups, there is a tiny typo:
```suggestion
      include 'python::pip::bootstrap'
```

Comment (feltra) [reply manifests/install.pp]: my bad, I didn't paid attention.

**Pull request merged**